tdrs-backend/docs/session-management.md

```diff
@@ -0,0 +1,11 @@
+# Session Management
+
+The requirement for this project is that users will be logged out of the system after 30 minutes of activity. This is our strategy to accomplish that.
+
+### Backend
+The backend will be the ultimate arbiter of session management. When the user logs in they will receive an HttpOnly cookie that is set to expire in 30 minutes. After that, with every interaction between the FE and BE, the BE will refresh the cookie, so it will extend the timeout time to another 30 minutes.
```

Comment:
Worth mentioning how this is configured? `SESSION_TIMEOUT` variable set in `common.py`, possible to override per-env?

Reply:
@alexsoble I have updated accordingly - good idea